lib/exceptions/WP_Auth0_EmailNotVerifiedException.php

```diff
@@ -1,13 +1,21 @@
 <?php
 
+/**
+ * Class WP_Auth0_EmailNotVerifiedException
+ */
 class WP_Auth0_EmailNotVerifiedException extends Exception {
 
 	public $userinfo;
 	public $id_token;
 
-	public function __construct( $userinfo, $id_token ) {
+	/**
+	 * WP_Auth0_EmailNotVerifiedException constructor.
+	 *
+	 * @param stdClass $userinfo - userinfo object returned from Auth0
+	 * @param string $id_token - DEPRECATED 3.4.1, should not be output in any template
```

Comment:
Move to a separate PR please

Reply:
@lbalmaceda - Will wait for direction on the other one so I can do them both. Should they each be in their own PR?

Reply:
Not necessary as long as the PR only deprecates stuff 👌